src/pages/Admin/Charity/EditProfile/index.tsx

```diff
@@ -34,7 +34,7 @@ function FormWithContext(props: EndowmentProfile) {
   const flatInitial: FlatFormValues = {
     name: props.name,
     categories_sdgs: props.categories.sdgs,
-    hq_country: props.hq.country || "",
+    hq_country: props.hq_country || "",
```

Comment:
no need for `|| ""` since `hq_country` is defined as `string`
```

type EndowmentBase = {
  hq_country: string;
  active_in_countries: string[];
```

if not sure if `hq_country` is defined should be updated 
```ts
type EndowmentBase = {
  hq_country?: string;
  active_in_countries: 
```

Reply:
The fetchoor Lambdas are set up in a way that will always return `hq_country` field. We can let the type stay as is.